draft-ietf-quic-recovery.md

```diff
@@ -268,6 +268,13 @@ packets. While this increases network load, it is considered safe, since the
 sender will still reduce its sending rate exponentially under persistent
 congestion ({{pto}}).
 
+## Handshake packets are not special
+
+TCP treats the loss of SYN or SYN-ACK packet as persistent congestion and
+reduces the congestion window to one packet ({{?RFC5681}}). QUIC treats
+loss of a packet containing handshake data the same as other losses and
+reduces the congestion window.  When the PTO timer expires during the
```

Comment:
This second sentence could be shortened.

```suggestion
reduces the congestion window to one packet ({{?RFC5681}}). QUIC treats
loss of a packet containing handshake data the same as other losses.
```

After all, you don't mention the recovery period or the potential persistent congestion piece here, but both apply.

Reply:
On the issue, the concern was about QUIC responding to loss during the handshake, so I thought the note about reducing CWND was relevant, but I agree that this is not the place to explain the full algorithms.